nntrainer/tensor/manager.cpp

```diff
@@ -918,6 +938,37 @@ void Manager::expand_lifespan(const std::string &name,
     enum_class_or<TensorLifespan>(tensor_lifespan_map[name], lifespan);
 }
 
+/**
+ * @brief     Create tensors with the given spec
+ */
+std::vector<Var_Grad *> Manager::requestAllocatedOutputsAsInputs(
+  const GraphNode &node, const std::vector<TensorDim> &inputs_dim,
```

Comment:
inputs_dim seems not needed in this function

Reply:
The signature is kept the same as `requestInput()` so as to merge them soon.